src/main/java/com/oltpbenchmark/api/StatementDialects.java

```diff
@@ -84,9 +85,9 @@ public String getSQLDialectPath(DatabaseType databaseType) {
 
         if (fileName != null) {
 
-            final String path = "benchmarks" + File.separator + workloadConfiguration.getBenchmarkName() + File.separator + fileName;
+            final String path = "/benchmarks/" + workloadConfiguration.getBenchmarkName() + "/" + fileName;
```

Comment:
Isn't this going to break on unix platforms?

Reply:
no, tested it (and you can see the results on unix travis tests [here](https://github.com/rusher/benchbase/actions/runs/1429002060))